waiter/integration/waiter/basic_test.clj

```diff
@@ -469,3 +470,28 @@
                    (map #(:git-version (waiter-settings % :cookies cookies)))
                    set
                    count))))))
+
+(deftest ^:parallel ^:integration-fast test-error-content-negotiation
```

Comment:
Please include a text that explicitly requests `text/plain`.

Nitpick: please use lower-case `accept` in the headers.